app/services/web_archive_glob.rb

```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 
-# Globs for web archive files (WARC, WACZ)
+# Globs for web archive files (WARC, ARC, WACZ)
 class WebArchiveGlob
   def self.warcs(directory)
-    Dir.glob("#{directory}/**/*.warc*")
+    Dir.glob("#{directory}/**/*.warc*") + Dir.glob("#{directory}/**/*.arc*")
```

Comment:
Maybe this is a six vs. half-dozen scenario but for you to consider:

```suggestion
Dir.glob("#{directory}/**/*.{arc,warc}*")
```

Reply:
I would consider. Thanks for the suggestion. Changed.